packages/common/client-utils/src/base64Encoding.ts

```diff
@@ -3,7 +3,8 @@
  * Licensed under the MIT License.
  */
 
-import { IsoBuffer } from "./indexNode.js";
+// Note: The exports map in package.json will correct this import to "./bufferNode.js" in node environments
+import { IsoBuffer } from "./bufferBrowser.js";
```

Comment:
Note: changed to 'bufferBrowser.js' instead of 'indexBrowser.js' to avoid import cycle.